docs/reference/activity-recognition.md

```diff
@@ -339,7 +339,7 @@ Inherited from [X3DLearner](#src.perception.activity_recognition.x3d.modules.x3d
 * **Fit model**.  
 
   ```python
-  from OpenDR.perception.activity_recognition.cox3d.cox3d_learner import X3DLearner
+  from OpenDR.perception.activity_recognition.cox3d.cox3d_learner import CoX3DLearner
```

Comment:
The package module is also `opendr`:
https://github.com/tasostefas/opendr_internal/blob/56306e7516402164c71ecdb7e0ab3ce951105f69/tests/sources/tools/perception/activity_recognition/cox3d/test_cox3d_learner.py#L20